ci-operator/jobs/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master-presubmits.yaml

```diff
@@ -73,6 +73,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-assisted,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
just to make sure
do you want it to always run on every PR of agent? or run only if someone does ``/test e2e-metal-assisted-day2``?
if the former, leave as is. if the later, change to ``always_run: false``

Reply:
hmm, marked it as optional, should it been auto generated accordingly? :/

Reply:
so optional meaning not required for merging the PR
but it's always auto-generatedso that the job is being triggered on every PR